### PR TITLE
Update pt_PT app translation for 3.9.0

### LINKED ISSUE #2699: Update pt_PT app translation for 3.9.0
Hi @Snayler,
We are getting ready for the 3.9.0 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **pt_PT** by **2022-07-22**.

Please note the new location of the translation files (.ts): They are now in src/translation. (Refactoring: move src/res/translation to src/translation #2576)

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-07-22

Suggested commit and PR message:
```
Update pt_PT app translation for 3.9.0

Fixes #2699
```

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by melcon**
Fixes #2699

### CODE REVIEW

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Review Checklist:

- [x] Translator listed in the `src/util.cpp` Not checking since long term translator
- [ ] Punctuation and spacing consistent
- [ ] Signal words consistent ("ASIO", "Buffer")
- [ ] App translations: No untranslated strings (`grep unfinished -5 src/translation/translation_$TRANSLATION*.ts`)
- [ ] App translations: Only a single `.ts` file checked in (`.qm` in addition is also OK)

**Comment by pljones** [comment]
@melcon if you're happy with these, we can get them merged.  Thanks for your help here.

File: src/translation/translation_pt_PT.ts [hunk 1]
```diff
@@ -1468,17 +1468,17 @@ O seu canal foi silenciado e foi activada a função &apos;Silenciar-me&apos;. P
...
-        <translation type="unfinished">Selecione o estilo a ser usado para os medidores de nível. As opções Barra (estreita) e LEDs (redondo, pequeno) aplicam-se apenas à mesa do mixer. Quando a Barra (estreita) é selecionada, os medidores de entrada são definidos como Barra l(larga). Quando LEDs (redondo, pequeno) é selecionado, os medidores de entrada são definidos como LEDs (redondo, grande). As opções restantes aplicam-se à mesa do mixer e aos medidores de entrada.</translation>
+        <translation>Selecione o estilo a ser usado para os medidores de nível. As opções Barra (estreita) e LEDs (redondo, pequeno) aplicam-se apenas à mesa do mixer. Quando a Barra (estreita) é selecionada, os medidores de entrada são definidos como Barra l(larga). Quando LEDs (redondo, pequeno) é selecionado, os medidores de entrada são definidos como LEDs (redondo, grande). As opções restantes aplicam-se à mesa do mixer e aos medidores de entrada.</translation>
```
Comment (ann0see) [inline src/translation/translation_pt_PT.ts:1471]: Is Barra l(larga) correct here? Looks a bit strange without space

**Comment by ann0see** [comment]
I will merge this in a few hours.

**Comment by ann0see** [comment]
Probably better tomorrow

File: src/translation/translation_pt_PT.ts (hunk 1 above)
Comment (melcon) [reply src/translation/translation_pt_PT.ts:1471]: Yeap, it is strange. That  **l** is a left over. Will be handled on the next PR

**Pull request merged**